addons/account/models/account.py

```diff
@@ -226,20 +225,53 @@ def load(self, fields, data):
         return rslt
 
     @api.multi
+    def _toggle_reconcile_to_true(self):
+        query = """
+            UPDATE account_move_line
+                SET reconciled = CASE
+                    WHEN debit = 0 AND credit = 0 AND amount_currency = 0
+                    THEN true
+                    ELSE false
+                    END,
+                full_reconcile_id = NULL,
+                amount_residual = (debit-credit),
+                amount_residual_currency = amount_currency
+            WHERE account_id IN %s
+        """
+        self.env.cr.execute(query, [tuple(self.ids)])
+
+    @api.multi
+    def _toggle_reconcile_to_false(self):
+        partial_lines_count = self.env['account.move.line'].search_count([
```

Comment:
I sugest a `RedirectWarning`, this also would reassure @fmdl

Reply:
Here is an example from the code, at hand:
```
if not rec_account and not pay_account:
    action = self.env.ref('account.action_account_config')
    msg = _('Cannot find a chart of accounts for this company, You should configure it. \nPlease go to Account Configuration.')
    raise RedirectWarning(msg, action.id, _('Go to the configuration panel'))
```

Reply:
I retract, I think the check the partial line check is sufficient, and I realise that otherwise the toggle off s quite none intrusive...